Customer feedback (#445)
foundriesio/docs: source/user-guide/custom-sota-client.rst

   https://github.com/foundriesio/aktualizr-lite/blob/master/include/aktualizr-lite/api.h

Using the custom-sota-client example
------------------------------------

We provide a `SOTA client`_ example in aktualizr-lite that serves as a great
starting place to experiment. The meta-lmp layer includes a recipe_ that
runs this example as the default SOTA client. Later, this serves as an example
to copy/paste into Factory specific recipe.

.. _recipe:
   https://github.com/foundriesio/meta-lmp/tree/master/meta-lmp-base/recipes-sota/custom-sota-client

.. _SOTA client:
   https://github.com/foundriesio/aktualizr-lite/tree/master/examples/custom-client-cxx

> Users can build this custom client into their LmP image with a simple

Reviewer: I think we've tried to stop using words like "simple", but I'll defer to katrina.

    echo 'SOTA_CLIENT = "custom-sota-client"' >> conf/machine/include/lmp-factory-custom.inc

Forking the custom-sota-client
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> The procedure for producing a factory-specific SOTA client can be by:

Reviewer 2: ```suggestion
Producing a factory-specific SOTA client can be done by:
```

 #. Creating a Git repository for your custom code. Copying the
    `examples/custom-client-cxx`_ directory is a good place to start.

>  #. Copying the `custom-sota-client`_ recipe from meta-lmp into

Reviewer 3: ```suggestion
 #. Copying the `custom-sota-client`_ recipe from `meta-lmp` into
```
